internal/task/package_tasks.go

```diff
@@ -115,6 +115,6 @@ func DefaultPackageTaskFactories() PackageTaskFactories {
 			pkgcataloging.DeclaredTag, pkgcataloging.DirectoryTag, pkgcataloging.InstalledTag, pkgcataloging.ImageTag, "binary"),
 		newSimplePackageTaskFactory(githubactions.NewActionUsageCataloger, pkgcataloging.DeclaredTag, pkgcataloging.DirectoryTag, "github", "github-actions"),
 		newSimplePackageTaskFactory(githubactions.NewWorkflowUsageCataloger, pkgcataloging.DeclaredTag, pkgcataloging.DirectoryTag, "github", "github-actions"),
-		newSimplePackageTaskFactory(sbomCataloger.NewCataloger, pkgcataloging.ImageTag, pkgcataloging.DeclaredTag, pkgcataloging.DirectoryTag, pkgcataloging.ImageTag, "sbom"), // note: not evidence of installed packages
+		newSimplePackageTaskFactory(sbomCataloger.NewCataloger, pkgcataloging.ImageTag, pkgcataloging.DeclaredTag, pkgcataloging.DirectoryTag, "sbom"), // note: not evidence of installed packages
```

Comment:
this is making a functional change from how this had been working; note that the SBOM cataloger had been included in image scans in the past: https://github.com/anchore/syft/blob/a16a4ad6c993aaaa1ec502667c215b0e8c8e51ff/syft/pkg/cataloger/cataloger.go#L62

That being said this change would close out https://github.com/anchore/syft/issues/1555 . (I think this is still being debated about, but we're looking to put this to rest over the next few working days)

The impact is that we should update the description of the PR and link the issue to this PR.

Reply:
I'm not sure about that! Does this need two instances of `pkgcataloging.ImageTag`? If so I think I'm not quite grasping what's happening here yet

Reply:
ah! disregard, I didn't see that it was there twice. Thanks!